db/schema.rb

```diff
@@ -54,8 +54,8 @@
     t.string   "experience"
     t.string   "interest"
     t.string   "school_id"
-    t.datetime "created_at",                                   :null => false
-    t.datetime "updated_at",                                   :null => false
+    t.datetime "created_at"
+    t.datetime "updated_at"
```

Comment:
Should this be so? I feel like not allowing null for those is good? Did the convention change?